### PR TITLE
Story CORE-500 | `/etc/hosts` cluster ip definition

### PR DESCRIPTION
**Pull request by nicholaspcr**
# Description

Kind: Story

Section: cliutils/viper | request.Send

Summary: Changed the http request made by the insprctl so that there is no longer a need to setup the /etc/hosts in your local machine

## Changelog

> List the main points of change

## Pay attention to

> Important parts of the code that require special attention

### CODE REVIEW

**Review by lucasABLima — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
...
 ## Release v0.1.1
 
+### #89 Story CORE-500 | removing inspr.com from local `/etc/hosts` 
+- fixes:
+    - added hostname variable to the Send func in the request pkg
+    - changed the git hook since `staticcheck` and `go vet` fails when looking only at a specific file, now looks at the entire repository folder
+    - changing minikube.md to replace `/etc/hosts` with a explanation on how to setup the serverip via the insprctl cli.
+
+---
+
```
Comment (franciscobonand) [inline CHANGELOG.md]: This should be outside of "Release v0.1.1"

**Review by LucasStarlingdePaulaSalles — APPROVED** [review]

File: pkg/rest/request/request.go [hunk 2]
```diff
@@ -124,6 +135,9 @@ func (c Client) handleResponseErr(resp *http.Response) error {
 			return err
 		}
 		return defaultErr
+	case http.StatusNotFound:
+		return ierrors.NewError().Message("Route not found").Build()
```
Comment (franciscobonand) [inline pkg/rest/request/request.go]: Capitalized error

**Review by franciscobonand — CHANGES_REQUESTED** [review]
Overall it's pretty nice. If it's in the scope of this story, it might be a good idea to set the default host as "inspr.com" in the CLI config list, so the user can change it if he wants (not needing to change a thing that is hardcoded).  
Also, if you do so, remember to check the docs to see if they need updates.

**Comment by nicholaspcr** [comment]
> Overall it's pretty nice. If it's in the scope of this story, it might be a good idea to set the default host as "inspr.com" in the CLI config list, so the user can change it if he wants (not needing to change a thing that is hardcoded).
> Also, if you do so, remember to check the docs to see if they need updates.

Given that some requests made by the Send function are not entirely on the cli i decided that it was more appropriate to leave it as a parameter of the request.Send method. The topic of leaving the user changing the hostname via `insprctl config` it doens't make much sense to me since this is default hostname used by the ingresses on Insprd and the UIDP, changing it would just make these services not accessible.

**Review by franciscobonand — APPROVED** [review]

**Pull request merged**